src/main/steps/applicant1/confirm-your-joint-application/content.ts

```diff
@@ -172,18 +171,178 @@ const en = ({ isDivorce, partner, userCase, isApplicant2, isJointApplication }:
+  caseReferenceHeading: 'Cyfeirnod yr achos',
+  caseReferenceValue: formattedCaseId(userCase.id),
+  line4: 'Ceisydd 1',
+  line5: getName(userCase, 'applicant1'),
+  line6: 'Ceisydd 2',
+  line7: getName(userCase, 'applicant2'),
+  subHeading2: `Ynghylch y ${isDivorce ? 'briodas' : 'bartneriaeth sifil'}`,
+  line8: `Mae’r manylion hyn wedi’u copïo yn uniongyrchol o’r dystysgrif ${
+    isDivorce ? 'priodas' : 'partneriaeth sifil'
+  } neu’r cyfieithiad o’r dystysgrif os nad yw’n Saesneg. Yr enwau ar y dystysgrif yw’r enwau roedd y ceiswyr yn eu defnyddio cyn y ${
+    isDivorce ? 'briodas' : 'bartneriaeth sifil'
+  }.`,
+  line9: `Rhwng pwy y mae’r ${isDivorce ? 'briodas ' : 'bartneriaeth sifil'}`,
+  line10: `${userCase.applicant1FullNameOnCertificate} a ${userCase.applicant2FullNameOnCertificate}
+      (fel y dangosir ar y dystysgrif ${isDivorce ? 'priodas' : 'partneriaeth sifil'})`,
+  line11: `Lle digwyddodd y ${isDivorce ? 'briodas ' : 'bartneriaeth sifil'}`,
+  line12: userCase.ceremonyPlace,
+  line13: `Dyddiad y ${isDivorce ? 'briodas ' : 'bartneriaeth sifil'}`,
+  line14: getFormattedDate(userCase.relationshipDate),
+  subHeading3: 'Pam gall y llys ddelio â’r achos (awdurdodaeth)',
+  line15: 'Mae gan lysoedd Cymru a Lloegr bŵer cyfreithiol (awdurdodaeth) i wrando’r achos hwn oherwydd:',
+  connectionBulletPoints:
+    userCase && userCase.connections
+      ? connectionBulletPointsSummarisedForAllUsers(userCase.connections, false, isDivorce, isJointApplication)
+      : [],
+  whatThisMeans: 'Beth mae hyn yn ei olygu',
+  jurisdictionsMoreDetails: {
+    part1: `Rhaid bod gan lysoedd Cymru a Lloegr y pŵer cyfreithiol (awdurdodaeth) i allu ${
+      isDivorce ? 'caniatáu ysgariad' : 'dod â phartneriaeth sifil i ben'
+    }.
+    Cadarnhaodd y ceiswyr bod y datganiad(au) cyfreithiol yn y cais yn berthnasol i naill ai un o’r ceiswyr neu’r ddau ohonynt. Mae pob datganiad cyfreithiol yn cynnwys rhai o’r cysylltiadau cyfreithiol canlynol â Chymru neu Lloegr, neu bob un ohonynt.`,
+    part2: jurisdictionMoreDetailsContent(userCase.connections, false, isDivorce, true).text,
+  },
+  subHeading4: 'Achosion llys eraill',
+  line16: `Mae’r llys angen gwybod am unrhyw achosion llys eraill sy’n ymwneud â’r ${
+    isDivorce ? 'briodas' : 'bartneriaeth sifil'
+  } a all effeithio ar bŵer cyfreithiol (awdurdodaeth) y llys.`,
+  line17: `${
+    userCase.applicant1LegalProceedings === YesOrNo.YES && userCase.applicant2LegalProceedings === YesOrNo.YES
+      ? `Mae Ceisydd 1 wedi rhoi manylion am achosion llys eraill sy’n ymwneud â’r ${
+          isDivorce ? 'briodas' : 'bartneriaeth sifil'
+        }:` +
+        '<br>' +
+        userCase.applicant1LegalProceedingsDetails +
+        '<br><br>' +
+        `Mae Ceisydd 2 wedi rhoi manylion am achosion llys eraill sy’n ymwneud â’r ${
+          isDivorce ? 'briodas' : 'bartneriaeth sifil'
+        }:` +
+        '<br>' +
```

Comment:
why do we add br tags in content, why not handle that in the template
I guess you're just replicating what's in the english version

Reply:
yep

Reply:
as mentioned before, i can clean up but i think theres alot to clean up so itll need its own ticket imo.